source/entities/Creature.cpp

```diff
@@ -559,6 +519,7 @@ void Creature::importFromPacket(ODPacket& is)
         mWeaponR = getGameMap()->getWeapon(tempString);
         OD_ASSERT_TRUE_MSG(mWeaponR != nullptr, "Unknown weapon name=" + tempString);
     }
+    setupDefinition(*getGameMap(), *ConfigManager::getSingleton().getCreatureDefinitionDefaultWorker());
```

Comment:
Creatures loaded from stream will be loaded before calling GameMap::createAllEntities. That means that calling it here may not be required (not sure, need testing).